nginx.tmpl

```diff
@@ -1,3 +1,6 @@
+{{/* Split the INCLUDED_HOSTS environment variable into an array */}}
+{{ $includedHosts := split "pma.localhost" "," }}
```

Comment:
I don't know much about this kind of code, but why is "pma.localhost" included?

Reply:
I guess the nginx.tmpl should be ignored in git, since it is now generated dynamically when running the start.sh script. Will update.